src/update_engine_ie9.jsx

```diff
@@ -64,8 +64,10 @@ export function update() {
  */
 export const requestAsyncUpdate = _.debounce(updateAfterDelay, 0);
 
-// update on window resize
-window.addEventListener('resize', _.debounce(requestAsyncUpdate, 16));
+if (typeof window !== 'undefined') {
```

Comment:
It currently is required, in [this line](https://github.com/byronmwong/react-flexbox-layout/pull/12/files#diff-1fdf421c05c1140f6d71444ea2b27638R10)